test/integration/context-based-restrictions.v1.test.js

```diff
@@ -18,7 +18,8 @@
 
 const { readExternalSources } = require('ibm-cloud-sdk-core');
 const { v4: uuidv4 } = require('uuid');
-const ContextBasedRestrictionsV1 = require('../../dist/context-based-restrictions/v1');
+// const ContextBasedRestrictionsV1 = require('../../dist/context-based-restrictions/v1');
+const ContextBasedRestrictionsV1 = require('@ibm-cloud/platform-services/context-based-restrictions/v1');
```

Comment:
@cam-d12 did you make this change or the generator code? The dist folder should have the latest locally built code. If you need to help with how to build the code into the dist folder, try the readme or reach out and I can try to help.